src/components/block/FooterCTACard/FooterCTACard.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Button from "../../inline/Button";
+import s from "./FooterCTACard.module.css";
+
+const FooterCTACard = ({ subtitle, body, buttonText }) => (
```

Comment:
I think it's a little bit confusing to call this prop the "subtitle" when there is no "title". Functionally, this prop acts as the title of this card, so it's probably better to name it "title".